### PR TITLE
Improve the documentation in the README

### PR DESCRIPTION
**Pull request by esolitos**
Changes:
- Adds a note about `--advertise-routes`
- Fixes the URL for `before-ssh.conf`
- Adds a `rm` in the Uninstalling notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ This was originally inspired by [lg](https://github.com/lg)'s [gist](https://gis
 
     The example below enables subnet routing for one subnet, enables use as an exit node (Tailscale 1.6+), and uses a one-off pre-auth key, which can be generated at https://login.tailscale.com/admin/authkeys
 
+    !! Remember to change `192.0.2.0/24` with the subnet(s) you *actually want to expose* to the tailnet.
```
Comment (jamesog) [inline README.md:47]: As the `!!` won't render to anything that looks particularly good, I'd suggest making this an emoji, e.g. `:information_desk_person:` (💁) or `:warning:` (⚠️). That should be more eye-catching to make sure people do change it.

**Review by jamesog — COMMENTED** [review]
Thanks for the contribution! Just one small comment.

File: README.md (hunk 1 above)
Comment (esolitos) [reply README.md:47]: Indeed, it should have been `:bangbang:` (‼️), but the warning icon might be even better. :)  

```suggestion
    :warning: Remember to change `192.0.2.0/24` with the subnet(s) you *actually want to expose* to the tailnet.
```

Comment (jamesog) [reply README.md:47]: I'm happy to go with any of these :-)

**Review by jamesog — APPROVED** [review]

**Comment by jamesog** [comment]
Thanks again!

**Pull request merged**